src/Sylius/Behat/Context/Setup/GeographicalContext.php

```diff
@@ -65,19 +73,45 @@ public function storeShipsTo($country1, $country2 = null, $country3 = null)
                 continue;
             }
 
-            $this->createCountryNamed(trim($countryName));
+            $this->countryRepository->add($this->createCountryNamed(trim($countryName)));
         }
     }
 
     /**
+     * @Given /^the store operates in "([^"]*)"$/
+     */
+    public function theStoreOperatesIn($countryName)
```

Comment:
With implementation in two lines, we can have one generic step here with shared storage setter, and then only disabling it in another one